### DISCUSSION TITLE
Possibility to add Samba during PiVPN installation

### DISCUSSION THREAD

**Discussion by Dretech** [pivpn/pivpn#1629] [Feature requests]
In the past view days I wrote a workshop about PiVPN for the Dutch Linux Magazine. During writing this workshop I concluded that installation of VPN clients can be a lot easier when Samba is active and enabled to share the home directory of the user that is used with PiVPN. With a file explorer on your desktop computer you can easily go to de Wireguard or OpenVPN configuration files stored on your PiVPN server.
My suggestion to improve PiVPN is to add to possibility to install Samba during the installation of PiVPN. The steps for integrating Samba in PiVPN are:

1. Install Samba: sudo apt install samba
2. Make Samba user. This user must have de same username as the user for which PiVPN is configured: sudo smbpasswd -a pivpnuser
3. Make home directories of Samba computer browse-able: Edit de smb.conf file. Under section [homes] set the value to 'yes' after  'browseable ='

**Comment by coolapso** [comment]
Don't think this is a feature we should add to PiVPN, However Its good material for the documentation page. 

If you want, I will be glad to review and accept a PR with that information here: https://github.com/pivpn/docs/tree/master/docs/guides
